modules/ocf_backups/files/upload-to-box

```diff
@@ -107,30 +109,43 @@ def get_access_token(creds):
 
 def upload_to_box(creds, quiet):
     """Uploads the archive to box at the configured location"""
-    lftp_cmd = '''
-        set ftps:initial-prot ""
-        set ftp:ssl-force true
-        set ftp:ssl-protect-data true
-        open {box_ftp}
-        user {email} {passwd}
-        mirror -R --no-overwrite --parallel={concurr} {archive} {folder}
-    '''.format(
-        box_ftp=BOX_FTP,
+
+    # curl takes credentials with a "netrc file"
+    netrc_creds = 'machine ftp.box.com login {email} password {passwd}'.format(
         email=creds['email'],
         passwd=creds['password'],
-        concurr=MAX_CONCURR_UPLOADS,
-        archive=ARCHIVE,
-        folder=FOLDER,
     )
+    netrc_file = NamedTemporaryFile(mode='w')
+    netrc_file.write(netrc_creds)
+    netrc_file.flush()
+
+    # Specify each file and its destination on the command line
+    # -T file1 ftp://.../destfolder/ -T file2 ftp://.../destfolder/ ...
+    file_args = list(chain.from_iterable(
```

Comment:
Nit: if you change the code below to
```
[
    'curl',
    '-1',
    '--disable-epsv',
    '--ftp-skip-pasv-ip',
    '--ftp-ssl',
    '--netrc-file', netrc_file.name,
    '--ftp-create-dirs',
    '--globoff',
    *file_args,
],
```

You don't need to wrap this in `list()`